index.d.ts

```diff
@@ -53,6 +53,13 @@ export interface Options extends Readonly<GlobOptions>, CpFileOptions {
 	*/
 	readonly flat?: boolean;
 
+	/**
+	Trim path from files being copied.
+
+	@default 0
+	*/
+	readonly up?: boolean;
```

Comment:
I don't think this is a good option name. Maybe `trimPathComponents` or something.